tests/unit/test_auth_middleware.py

```
@@ -0,0 +1,330 @@
+
+
+def create_test_token(
+    email: str = "test@example.com",
+    exp_delta: timedelta = timedelta(hours=1),
+    iat_delta: timedelta = timedelta(seconds=0),
+    audience: str = TEST_AUDIENCE,
+    issuer: str = f"https://{TEST_TEAM_DOMAIN}",
+    kid: str = "test-key-1",
+) -> str:
+    """Create a test JWT token."""
+    now = datetime.utcnow()
```

Ruff (0.14.8)

`datetime.datetime.utcnow()` used

(DTZ003)